src/docfx/config/Config.cs

```diff
@@ -10,8 +10,13 @@ namespace Microsoft.Docs.Build
 {
     internal sealed class Config
     {
-        private static readonly string[] s_defaultInclude = new[] { "docs/**/*.{md,yml,json}" };
-        private static readonly string[] s_defaultExclude = new[] { "_site/**/*", "localization/**/*" };
+        private static readonly string[] s_defaultInclude = new[] { "**/*.{md,yml,json}" };
+        private static readonly string[] s_defaultExclude = new[]
+        {
+            "_site/**",             // Default output location
+            "localization/**",      // Localization file when using folder convention
+            "**/includes/**",       // Default convention for reusable markdown tokens
```

Comment:
>includes [](start = 16, length = 8)

do we also want to add `include`, `token`, `tokens` and like `README.md`, `CONTRIBUTING.md`, `LICENSE`

Reply:
solved with `defaultExclude`